index.bs

```diff
@@ -1562,6 +1649,16 @@ ReadableStreamBYOBReader(<var>stream</var>)</h4>
   1. Return *true*.
 </emu-alg>
 
+<h4 id="is-readable-stream-default-reader-asynciterator-prototype">
+aoid="IsReadableStreamDefaultReaderAsyncIterator" nothrow>IsReadableStreamDefaultReaderAsyncIterator (
+ <var>argument</var> )</h4>
+
+<emu-alg>
+  1. If Type(_argument_) is not Object, return *false*.
+  1. If _argument_ does not have a [[Reader]] and [[PreventCancel]] internal slots, return *false*.
```

Comment:
@TimothyGu previously [reported this](https://github.com/whatwg/streams/pull/950#discussion_r212138871), but it hasn't been tackled yet:
> The "a" seems misplaced.